hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobRepository.java

```diff
@@ -102,6 +103,12 @@
      */
     public static final String JOB_RECORDS_MAP_NAME = INTERNAL_JET_OBJECTS_PREFIX + "records";
 
+    /**
+     * Name of internal IMap which stores {@link JobRecord}s.
```

Comment:
Wrong comment. Also I'd name the map "results_metrics", because they are related.